syntax = "proto3";
package proxy;

// The RPC service
service ProxyService {
  // maps to /ping HTTP endpoint
  rpc Predict (PredictRequest) returns (PredictResponse);
}

// request message
message PredictRequest {
  string json_request = 1;
}

// response message
message PredictResponse {
  string json_response = 1;
}

